crates/ruff/src/rules/flake8_errmsg/rules.rs

```diff
@@ -7,6 +7,51 @@ use crate::registry::{Diagnostic, Rule};
 use crate::violation::Violation;
 
 define_violation!(
+    /// ## What it does
+    /// Checks for raw usage of a string literal in Exception raising.
```

Comment:
I know that the rule is currently named RawStringInException but I think it should be renamed from raw to direct since raw string usually refers to strings such as `r"this"`.

So "raw usage" should be reworded here. Perhaps:

Checks for the direct supplying of a string literal to an exception constructor.